public/components/marketing-cards/29.html

```diff
@@ -0,0 +1,17 @@
+<div
+  class="flex w-full aspect-[3/2] flex-col gap-4 overflow-hidden rounded-lg bg-gray-100 shadow-md transition-all hover:scale-[101%] hover:shadow-lg"
```

Comment:
Using `aspect` on the card itself doesn't feel right.

Reply:
The aspect ratio maintains the size of the card so that there is enough room for an absolute positioned image